src/css/_page.scss

```diff
@@ -117,3 +117,58 @@ article {
 #content-wrap {
   padding-bottom: var(--footer-height);
 }
+
+/* Styles for blockquote alerts, aka admonitions or callouts */
+// Source: https://github.com/KKKZOZ/hugo-admonitions
```

Comment:
Any specific reason not to use the module instead of c/p?

Reply:
@Wryhder

Reply:
That would be preferable than duplicate functionality IMHO.